Packs/Hackuity/Integrations/Hackuity/README.md

```diff
@@ -0,0 +1,175 @@
+| Hackuity.Vulnerabilities.Findings.Ignored | string | The number of ignored findings on this vulnerability | 
+| Hackuity.Vulnerabilities.VulnType.Id | string | The ID of the vulnerability type | 
+| Hackuity.Vulnerabilities.VulnType.Name | string | The name of the vulnerability type | 
+
+### hackuity-search-provider-vulnerabilities
+***
+Search for provider vulnerabilities in Hackuity.
+
+
+#### Base Command
+
+`hackuity-search-provider-vulnerabilities`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| asset_name | The name of the asset. | Optional | 
+| asset_type | The type of the asset if the asset name is specified (by default, restricts to IPs &amp; domains). | Optional | 
+| attribute | An attribute value. | Optional | 
+| cvss_min | The minimum CVSS (included). | Optional | 
+| cvss_max | The maximum CVSS (excluded). | Optional | 
+| limit | The maximum number of items to return. Default is 20. | Optional | 
+| trs_min | The minimum TRS (included). | Optional | 
+| trs_max | The maximum TRS (excluded). | Optional | 
+| vuln_type | The vulnerability type (ID). | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Hackuity.Vulnerabilities.Attributes | string | The attributes of the vulnerability | 
+| Hackuity.Vulnerabilities.Score.CVSS | string | The CVSS of the vulnerability | 
+| Hackuity.Vulnerabilities.Score.TRS | string | The TRS of the vulnerability | 
+| Hackuity.Vulnerabilities.Description | string | The description of the vulnerability | 
+| Hackuity.Vulnerabilities.ID | string | The ID of the vulnerability | 
+| Hackuity.Vulnerabilities.Seen.First | date | The date of the first time the vulnerability has been seen | 
+| Hackuity.Vulnerabilities.Findings.Total | string | The total number of findings on this vulnerability | 
+| Hackuity.Vulnerabilities.Findings.Open | string | The number of open findings on this vulnerability | 
+| Hackuity.Vulnerabilities.Findings.Closed | string | The number of closed findings on this vulnerability | 
+| Hackuity.Vulnerabilities.Findings.Ignored | string | The number of ignored findings on this vulnerability | 
+| Hackuity.Vulnerabilities.VulnType.Id | string | The ID of the vulnerability type | 
+| Hackuity.Vulnerabilities.VulnType.Name | string | The name of the vulnerability type | 
+
+### hackuity-dashboard-widgets
+***
+List the widgets in the default dashboard of the user.
+
+
+#### Base Command
+
+`hackuity-dashboard-widgets`
+#### Input
+
+There are no input arguments for this command.
+
+#### Context Output
+
+There is no context output for this command.
+### hackuity-dashboard-data
+***
+Get the data of a dashboard widget
+
+
+#### Base Command
+
+`hackuity-dashboard-data`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| widget_id | The ID of the widget. | Required | 
+
+
+#### Context Output
+
+There is no context output for this command.
```

Comment:
why dont you have a command example with returned data to any of the commands? it should be provided.

Reply:
Done

Reply:
where? the file is not updated